Makefile

```diff
@@ -24,26 +29,73 @@ test:
 	go test ./op-chain-ops/...
 	go test ./kroma-validator/...
 	go test ./op-e2e/... -timeout 30m # requires a minimum of 30min in a CI
-	yarn test
+	pnpm test
 .PHONY: test
 
-lint:
-	@go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.46.2 \
-	&& golangci-lint run
-.PHONY: lint
+golang-docker:
+	# We don't use a buildx builder here, and just load directly into regular docker, for convenience.
+	GIT_COMMIT=$$(git rev-parse HEAD) \
+	GIT_DATE=$$(git show -s --format='%ct') \
+	IMAGE_TAGS=$$GIT_COMMIT,latest \
+	docker buildx bake \
+			--progress plain \
+			--load \
+			-f docker-bake.hcl \
+			op-node op-batcher op-proposer op-challenger
+.PHONY: golang-docker
+
+submodules:
+	# CI will checkout submodules on its own (and fails on these commands)
+	if [ -z "$$GITHUB_ENV" ]; then \
+		git submodule init; \
+		git submodule update --recursive; \
+	fi
+.PHONY: submodules
 
 bindings:
-	make -C ./op-bindings
-.PHONY: op-bindings
+	make -C ./kroma-bindings
+.PHONY: bindings
```

Comment:
```suggestion
.PHONY: kroma-bindings
```

Reply:
It would be better to delete this line. I will modify next time.